web/react/stores/preference_store.jsx

```diff
@@ -122,5 +122,6 @@ class PreferenceStoreClass extends EventEmitter {
 }
 
 const PreferenceStore = new PreferenceStoreClass();
+PreferenceStore.setMaxListeners(0);
```

Comment:
But what makes them components is that they are independent and don't need a fixed parent to feed them data updates. That way IMO this - subcribe the posts to stores - is more react-like...